### PR TITLE
test(view): add tests to terminalview and sdlview

### PR DESCRIPTION
**Pull request by AtomToast**
based on #35 
Fix tests for terminalview
Add tests to sdlview
small refactor to sdlview

### CODE REVIEW

**Comment by anarchuser** [comment]
should travis run by now?

File: test/view/sldview/sdlview.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        Point position = {0,0};
+        Point expectedPosition = {0,0};
+        REQUIRE(
+                sdlview.getRelativePosition(position, worldWidth, worldHeight, windowWidth, windowHeight) == expectedPosition
+                );
+    }
+
+    SECTION("The conversion works successfully at maximum position") {
+        Point position = {1000,1000};
+        Point expectedPosition = {1080,1080};
```
Comment (anarchuser) [inline test/view/sldview/sdlview.cpp:23]: you can use an even shorter notation:
```Point position (1000, 1000);```

**Comment by AtomToast** [comment]
> should travis run by now?

no, I didn't change anything from where we left off and I have absolutely no idea what to try next to get it working

File: test/view/sldview/sdlview.cpp (hunk 1 above)
Comment (AtomToast) [reply test/view/sldview/sdlview.cpp:23]: I mean it's not much shorter but I guess I can use it

Comment (anarchuser) [reply test/view/sldview/sdlview.cpp:23]: you don't have to but knowing that it also works is good

**Review by anarchuser — APPROVED** [review]

**Pull request merged**